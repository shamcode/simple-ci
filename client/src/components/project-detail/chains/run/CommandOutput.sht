<pre class="command-output">
    {{output}}
</pre>