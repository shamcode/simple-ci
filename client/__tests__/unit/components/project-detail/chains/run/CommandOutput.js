// eslint-disable-next-line max-len
import CommandOutput from '../../../../../../src/components/project-detail/chains/run/CommandOutput.sht';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const meta = renderer( CommandOutput, {} );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
