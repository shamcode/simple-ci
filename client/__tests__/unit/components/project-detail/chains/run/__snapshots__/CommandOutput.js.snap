// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "Options": Object {
    "ID": "component",
  },
  "html": "<pre class=\\"command-output\\"></pre>",
}
`;
